nestkernel/spike_data.h

```diff
@@ -269,6 +281,10 @@ class OffGridSpikeData : public SpikeData
   double get_offset() const;
 };
 
+//!< check legal size
```

Comment:
Doxygen documentation placed before the thing they document should start with `//!` only.